packages/d/droidcam/package.yml

```diff
@@ -0,0 +1,41 @@
+name       : droidcam
+version    : 2.1.2
+release    : 1
+source     :
+    - https://github.com/dev47apps/droidcam/archive/refs/tags/v2.1.2.tar.gz : db65ccabc2f54bcd581963e2c36c56a611eea45e5332a39d24440b0966a704b4
+homepage   : http://www.dev47apps.com/
+license    : GPL-2.0-or-later
+component  : multimedia.video
+summary    : GNU/Linux client for DroidCam
+description: |
+    DroidCam turns your phone/tablet into a webcam for your PC. Use it with chat programs like Zoom, MS Teams, and Skype.
+
+    Installation Guide : Please install `v4l2loopback-current` if you are on default kernel(`linux-current`) or `v4l2loopback` if you are on LTS kernel(`linux-lts`).
```

Comment:
We don't put instructions in the description, this would be better in the help center.

Reply:
How do i go about this? Remove instructions for now? Add instruction later when it is added in the help center? I still need to learn about making PR in help center.

Reply:
Probably drop the instructions for now, I can merge without them. Perhaps add something in [this folder](https://github.com/getsolus/help-center-docs/tree/master/docs/user/hardware)?